scripts/base/protocols/ntlm/main.zeek

```diff
@@ -35,6 +35,9 @@ export {
 		## Internally used field to indicate if the login attempt
 		## has already been logged.
 		done: bool  &default=F;
+
+		server_challenge: string &log &optional;
+		ntlmv2hash: string &log &optional;
```

Comment:
This will add `server_challenge` and `ntlmv2hash` to the default `ntlm.log`. I'm tempted to say they aren't needed/interesting in all deployments. `ntlmv2hash` ends-up containing redundant info (username, domain and the challenge).

With your changes adding the `challenge` and `response` fields to the respective NTLM types, the log can be extended via an external package or just additional script. It does not need to be part of Zeek base. Adding them by default requires deployments that are not interesting in them to opt-out. I'd go for opt-in for these.

Reply:
Hmm, how about having it as an optional script like in kerberos ticket-logging ?

Reply:
For now I removed my changes from the ntlm script